docs/commands/fsmove.md

```diff
@@ -0,0 +1,33 @@
+# `fsmove` command
+
+Note: The implementations for `fscopy` and `fsmove` are exactly the same. The only difference is that `fsmove` will remove the source after a successful copy.
+
+The `fsmove` command works like the Unix `mv` or `rsync` binaries. It allows moving either single entries or whole folders in and out of the gopass mount.
+
+If the source is a directory, the source directory is re-created at the destination if no trailing slash is found. Otherwise the contained secrets are placed into the destination directory (similar to what `rsync` does).
+
+## Synopsis
+
+```
+# Decrypt the secret "leaf" and place in /home/user/.secrets directory
+$ gopass fsmove path/to/leaf /home/user/.secrets
```

Comment:
I think you meant fscopy here, not fsmove?
IIUC fsmove would delete the secrets you're moving into the fs.